tools/test-proxy/Azure.Sdk.Tools.TestProxy/CommandParserOptions/StartOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using CommandLine;
+using System.Collections.Generic;
+
+namespace Azure.Sdk.Tools.TestProxy.CommandParserOptions
+{
+    [Verb("start", isDefault: true, HelpText = "Start the TestProxy.")]
```

Comment:
For a wider audience, this is the default action of the test-proxy executable. There may be one or multiple test-proxies active on a given machine at a time. So someone would say `test-proxy start <args>` or `test-proxy serve <args>` to start a server instance.

I prefer `serve`, but my opinions on stuff like this can be wonky. Putting it to a poll of the team.

Reply:
Like this comment if you prefer `test-proxy start`

Reply:
Like this comment if you prefer `test-proxy serve`

Reply:
I think serve is a little more descriptive but the first command word I would think to try with any new tool would be start, hence my vote. That way people are more likely get to the right functionality or sub-docs faster.

Reply:
@scbedd : Does the command block?

Reply:
@richardpark-msft I don't really understand the question. It starts it as a server, so you'll see the asp.net startup output and then whatever informational logging as requests hit it. A ctrl-c in that session will kill it.

Reply:
The other commands will have to do with asset-sync functionality, and will be run to completion kinda things. Those can possibly block on user input or the like.

Reply:
`start` is straightforward and clear to me. I need more explanations when I saw `serve`.

Reply:
> A ctrl-c in that session will kill it.

@scbedd - that's what I wanted to know. Does it just start something and terminate, or does it continue to run. Voted for 'serve'.

Reply:
@scbedd +1 to @benbp comment, I personally don't like the word `serve` in thix context. In my mind, the first command that I would ever try when using a new tool would be `start`, `init`, or `run`

Reply:
I think enough folks have responded to form a consensus here. Thanks for your input! We'll go with `start`.

Reply:
Given @richardpark-msft's question about it being blocking I also tend to lean towards `serve` over  `start`. To me when I think about start I wouldn't expect it to block. FWIW other tools like jekyll also use the term `serve`, see https://jekyllrb.com/docs/usage/.

Reply:
@JonathanCrd's suggestion of 'run' may have some legs. Docker run, dotnet run, go run, etc.

Also you can always support all the words as command aliases. I don't see a big downside there as long as you keep it small-ish, and pick a primary name for the command and print it out (git has a similar approach for deprecated aliases).